src/test/java/uk/gov/hmcts/cmc/claimstore/events/response/DefendantResponseCitizenNotificationsHandlerTest.java

```diff
@@ -52,4 +55,19 @@ public void notifyDefendantResponseSendsNotificationsToClaimant() throws Notific
             eq("claimant-response-notification-" + RESPONSE_EVENT.getClaim().getReferenceNumber())
         );
     }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void throwExceptionWhenResponseNotPresent() throws IllegalArgumentException {
```

Comment:
minor : you don't need the `throws IllegalArgumentException`

Reply:
Same applies for throwExceptionResponseEventIsGeneratedWithNullClaim where null pointer is thrown. Removing both.